### ISSUE TITLE
CCL Caught an exception rendering a block

### ISSUE THREAD

**Issue by aaronhowser1** [RS485/LogisticsPipes#1449]
```java
[06:07:59] [Client thread/INFO] [minecraft/GuiNewChat]: [CHAT] CCL Caught an exception rendering a block. See the log for info.
[06:07:59] [Chunk Batcher 0/ERROR] [CodeChickenLib]: 
 CCL has caught an exception whilst rendering a block
  BlockPos:      x:1090, y:4, z:149
  Block Class:   class logisticspipes.blocks.LogisticsSolidBlock
  Registry Name: logisticspipes:crafting_table
  Metadata:      0
  State:         logisticspipes:crafting_table[active=true,connection_0=true,connection_1=true,connection_2=true,connection_3=true,connection_4=true,connection_5=true,rotation=0]
 Tile at position
  Tile Class:    class logisticspipes.blocks.crafting.LogisticsCraftingTableTileEntity
  Tile Id:       logisticspipes:logistics_crafting_table
  Tile NBT:      {fuzzyFlags:[{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b}],placedBy_id:"13027fdb-aac9-491b-9d93-c7e2f791d846",rotation:2,outputFuzzyFlags:{ignore_dmg:0b,ignore_nbt:0b,use_od:0b,use_category:0b},invitems:[],matrixitems:[],x:1090,matrixitemsCount:9,y:4,ForgeCaps:{"nuclearcraft:capability_default_radiation_resistance":{radiationResistance:0.0d}},z:149,placedBy_name:"1aaron5",id:"logisticspipes:logistics_crafting_table",invitemsCount:18}
You can turn off player messages in the CCL config file.

net.minecraft.util.ReportedException: Tesselating block in world
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:95) ~[bvm.class:?]
	at codechicken.lib.render.block.CCBlockRendererDispatcher.renderBlock(CCBlockRendererDispatcher.java:82) [CCBlockRendererDispatcher.class:?]
	at net.minecraft.client.renderer.chunk.RenderChunk.rebuildChunk(RenderChunk.java:200) [bxr.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.processTask(SourceFile:100) [bxn.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:43) [bxn.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
Caused by: java.util.ConcurrentModificationException
	at java.util.ArrayList$Itr.next(ArrayList.java:865) ~[?:1.8.0_212]
	at net.minecraft.client.renderer.BlockModelRenderer.handler$zbj000$beforeRenderModel(BlockModelRenderer.java:558) ~[bvo.class:?]
	at net.minecraft.client.renderer.BlockModelRenderer.renderModel(BlockModelRenderer.java) ~[bvo.class:?]
	at net.minecraft.client.renderer.BlockModelRenderer.renderModel(BlockModelRenderer.java:38) ~[bvo.class:?]
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:80) ~[bvm.class:?]
	... 5 more

```

![image](https://user-images.githubusercontent.com/15698722/81059789-c3d55c80-8e96-11ea-8169-b0b3d4ec3357.png)

**Comment by Heldenkrieger01** [comment]
I have also experienced this. This only happened once to me. When I (when the block was bugged and invisible and the CCL message appeared) tried to break it, my client got disconnected. Subsequently, I experienced very weird rendering bugs in the title menu (picture and text offset from eachother, sorry no screenshot of that). The next time I logged on, all was back to normal and the Compiler was still there.

[Here](https://pastebin.com/t25mAC7T) is the full log from the disconnect until I stopped the client in the menu.
